cni-plugins/sriov-passthrough-cni/plugin/sriov-passthrough-cni

```diff
@@ -38,6 +40,12 @@ function setns_sriov_pfs() {
   for pf in "${sriov_pfs[@]}"; do
     local pf_name="${pf%%/device/*}"
     pf_name="${pf_name##*/}"
+
+    if reserved_pf "${pf_name}"; then
+      log "PFs ${pf_name} is system reserved, skipping.."
```

Comment:
I don't think we need this log, it will print it a lot and it isn't helpful

Reply:
In the case of the new cluster - this will be 4 extra log lines which will provide useful information and only prints if we have `SYSTEM_RESERVED_PFS` set.

Reply:
for each CNI run / each job which is a lot
open please the file and see why it is not needed imo
It doesn't add info as this is the same thing again and again

Reply:
Yes makes sense to me now - we don't need to log this each time a test pod is deployed.

Reply:
Is there a shortage of storage for logs? if we do, we have bigger problems then this log message..
If not, I don't think we need to be hard on this log message, which could eventually help with debugging.

Note that removing this log message will make debugging harder when something goes wrong.

Reply:
How about printing the reserved interface names (the env var value) before starting the loop?

This way it won't repeat if there is more than one iteration.
We will see this message per ADD command.

Reply:
We don't need it, it is fixed in the PR of the daemonset